portal-api/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "dtfs-portal-api",
```

This is the only one with `dtfs` in the name as a suffix
```suggestion
  "name": "portal-api",
```

---

Have updated.